### PR TITLE
add Datadog.dd-trace-dotnet 1.27.0

### PR DESCRIPTION
**Pull request by lucaspimentel**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15846)

### CODE REVIEW

File: manifests/d/Datadog/dd-trace-dotnet/1.27.0/Datadog.dd-trace-dotnet.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: Datadog.dd-trace-dotnet
+PackageVersion: 1.27.0
+PackageLocale: en-US
+Publisher: Datadog, Inc.
+PublisherUrl: https://docs.datadoghq.com/
+PublisherSupportUrl: https://www.datadoghq.com/support/
+PrivacyUrl: https://www.datadoghq.com/legal/privacy/
+Author: Datadog
+PackageName: Datadog APM .NET Tracer
```
Comment (ItzLevvie) [inline manifests/d/Datadog/dd-trace-dotnet/1.27.0/Datadog.dd-trace-dotnet.locale.en-US.yaml:10]:
```suggestion
PackageName: Datadog .NET Tracer 64-bit
```
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/d/Datadog/dd-trace-dotnet/1.27.0/Datadog.dd-trace-dotnet.locale.en-US.yaml (hunk 1 above)
Comment (lucaspimentel) [reply manifests/d/Datadog/dd-trace-dotnet/1.27.0/Datadog.dd-trace-dotnet.locale.en-US.yaml]: Thanks, I removed "APM" from the name to better match the msi product name, but we cannot add "64-bit". We actually have two msi installers (x64 and x86) with different `ProductName` (one says "64-bit" and the other "32-bit"). For now we're only listing the x64 installer, but the goal is to list both installers once that is supported by winget, so the top-level `PackageName` in the manifest cannot match the different `ProductName` in the two msi.

- winget `PackageName`: Datadog .NET Tracer
  - x64 msi `ProductName`: Datadog .NET Tracer 64-bit
  - x86 msi `ProductName`: Datadog .NET Tracer 32-bit

Comment (ItzLevvie) [reply manifests/d/Datadog/dd-trace-dotnet/1.27.0/Datadog.dd-trace-dotnet.locale.en-US.yaml]: That's fine :)

**Comment by lucaspimentel** [comment]
CI seems stuck. Will try closing and re-opening.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**